src/daemon/custom_image_host.cpp

```diff
@@ -68,16 +68,16 @@ const QMap<QString, CustomImageInfo> multipass_image_info{
 const QMap<QString, CustomImageInfo> snapcraft_image_info{
     {{"ubuntu-16.04-minimal-cloudimg-amd64-disk1.img"},
      {"https://cloud-images.ubuntu.com/minimal/releases/xenial/release/",
-      {"core", "core16"},
```

Comment:
Please do not remove core, it is a valid use case:

- as a trigger for Snapcraft 3.0 functionality
- as the default when using multipass without a base (e.g.; OS X).

It is also documented here https://snapcraft.io/docs/base-snaps

Reply:
Of course. I confused the two… guess why :P

Fixed.